charts/spire/charts/spire-server/README.md

```diff
@@ -80,6 +84,10 @@ A Helm chart to install the SPIRE server.
 | nodeAttestor.k8sPsat.serviceAccountAllowList | list | `[]` |  |
 | nodeSelector | object | `{}` |  |
 | notifier.k8sbundle.namespace | string | `""` | Namespace to push the bundle into, if blank will default to SPIRE Server namespace |
+| persistence.accessMode | string | `"ReadWriteOnce"` |  |
+| persistence.enabled | bool | `true` |  |
+| persistence.size | string | `"1Gi"` |  |
+| persistence.storageClass | string | `nil` |  |
```

Comment:
What do you think of keep this together with the rest of the datastore config?

Reply:
It doesn't have a 1:1 relationship with dataStore. 

Its potentially used by sqlite3 for spire-server, by the disk KeyStore, and/or by Tornjak.

So when mysql/postgresql is used that has its own storage, it still may be needed.

Reply:
See https://github.com/spiffe/helm-charts/pull/211
As its not merged, and this kind of depends on it, some of the changes appear here too.